packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+| limit       | Specifies the number of results for a page (default is 20)     | Query        | Number     | FALSE    |
+| roleId      | `id` of a Role                                                 | URL          | String     | TRUE     |
+
+Treat the page cursor as an opaque value: obtain it through the next link relation. See [Pagination](/docs/reference/api-overview/#pagination).
+
+###### Response parameters
+
+Array of catalog Apps
+
+If the Role isn't scoped to specific Apps in the catalog, an empty array `[]` is returned.
+
+###### Request example
+
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3/roles/IFIFAX2BIRGUSTQ/targets/catalog/apps"
+```
+
+###### Response example
+
+The example shows one application and one instance. Note the response for instances has an `id` field.
+
+```json
+[
+    {
+        "name": "facebook",
+        "displayName": "Facebook",
+        "description": "Giving people the power to share and make the world more open and connected.",
+        "status": "ACTIVE",
+        "lastUpdated": "2017-07-19T23:37:37.000Z",
+        "category": "SOCIAL",
+        "verificationStatus": "OKTA_VERIFIED",
+        "website": "http://www.facebook.com/",
+        "signOnModes": [
+            "BROWSER_PLUGIN"
+        ],
+        "_links": {
+            "logo": [
+                {
+                    "name": "medium",
+                    "href": "http://${yourOktaDomain}/assets/img/logos/facebook.e8215796628b5eaf687ba414ae245659.png",
+                    "type": "image/png"
+                }
+            ],
+            "self": {
+                "href": "http://${yourOktaDomain}/api/v1/catalog/apps/facebook"
+            }
+        }
+    },
+    {
+        "name": "24 Seven Office 0",
+        "status": "ACTIVE",
+        "id": "0oasrudLtMlzAsTxk0g3",
+        "_links": {
+            "self": {
+                "href": "http://${yourOktaDomain}/api/v1/apps/0oasrudLtMlzAsTxk0g3"
+            }
+        }
+    }
+]
+```
+
+#### Add an App target to an App administrator role
+
+##### Add an App target to an App administrator role given to a User
+
+<ApiOperation method="put" url="/api/v1/users/${userId}/roles/${roleId}/targets/catalog/apps/${appName}" />
+
+Adds an App target for an `APP_ADMIN` Role assigned to a User
+
+When you add the first App target, you reduce the scope of the role assignment. The Role no longer applies to all App targets, but applies only to the specified target.
+
+Adding an App target overrides any existing App Instance targets of the App. For example, if someone was assigned to administer a specific Facebook instance, calling this endpoint with `facebook` for `appName`, would make them administrator for all Facebook instances.
```

Comment:
calling this endpoint > a call to this endpoint
remove comma before "would"
would make them administrator for > would make that person the administrator for